packages/babel-preset-pwa-kit/package.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "babel-preset-pwa-kit",
+    "version": "2.0.0-dev.6",
+    "engines": {
+      "node": "^12.0.0 || ^14.0.0",
+      "npm": "^6.14.4 || ^7.0.0 || ^8.0.0"
+    },
+    "description": "Babel preset used by PWA Kit",
+    "repository": {
+      "type": "git",
+      "url": "git+https://github.com/SalesforceCommerceCloud/pwa-kit.git"
+    },
+    "homepage": "https://github.com/SalesforceCommerceCloud/pwa-kit/tree/develop/packages/babel-preset-pwa-kit#readme",
+    "bugs": {
+      "url": "https://github.com/SalesforceCommerceCloud/pwa-kit/issues"
+    },
+    "license": "SEE LICENSE IN LICENSE",
+    "author": "dev@mobify.com",
+    "main": "index.js",
+    "scripts": {
+      "lint": "npm run lint:js",
+      "lint:fix": "npm run lint:js -- --fix",
+      "lint:js": "cross-env ../../node_modules/.bin/eslint \"**/*.{js,jsx}\""
+    },
+    "dependencies": {
```

Comment:
We need to include these two plugins as dependencies:
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/88ad67fbfdccb9f19b33681df2c5fa4ef07e0d7d/packages/babel-preset-pwa-kit/index.js#L25-L26